.dockerignore

```diff
@@ -1 +1 @@
-**/*/Dockerfile
```

Comment:
Is this entire commit (https://github.com/e-mission/op-admin-dashboard/pull/7/commits/185e5e94c44217b1b83ed6ff6f0cc80fff163721#diff-2f754321d62f08ba8392b9b168b83e24ea2852bb5d815d63e767f6c3d23c6ac5L1) just adding a final line to each file?

Reply:
I thought there were other changes in this commit, but I didn't add them by mistake, and they are in the other commit.